diff --git a/node_modules/typescript/lib/typescript.js b/node_modules/typescript/lib/typescript.js
index 2643aa1..c3a3c2c 100644
--- a/node_modules/typescript/lib/typescript.js
+++ b/node_modules/typescript/lib/typescript.js
@@ -45059,7 +45059,10 @@ function tryLoadModuleUsingBaseUrl(extensions, moduleName, loader, state) {
   if (state.traceEnabled) {
     trace(state.host, Diagnostics.Resolving_module_name_0_relative_to_base_url_1_2, moduleName, baseUrl, candidate);
   }
-  return loader(extensions, candidate, !directoryProbablyExists(getDirectoryPath(candidate), state.host), state);
+  const res = loader(extensions, candidate, !directoryProbablyExists(getDirectoryPath(candidate), state.host), state);
+  if (res) {
+    throw new Error("TS_FIX_BASEURL_SIGNAL")
+  }
 }
 function resolveJSModule(moduleName, initialDir, host) {
   const { resolvedModule, failedLookupLocations } = tryResolveJSModuleWorker(moduleName, initialDir, host);
